app/actions/websocket/channels.ts

```diff
@@ -236,3 +227,23 @@ export function handleUpdateMemberRoleEvent(msg: WebSocketMessage) {
     };
 }
 
+export function fetchChannelAndAddToSidebar(channelId: string, type?: string) {
+    return async (dispatch: DispatchFunc, getState: GetStateFunc): Promise<ActionResult> => {
+        const channelActions = await fetchChannelAndMyMember(channelId);
+        let channel;
```

Comment:
This is referring to the action type, perhaps rename to `hasCannel` or something similar?